CMakeLists.txt

```diff
@@ -155,4 +150,4 @@ target_link_libraries(uprotocolsdk_shared Threads::Threads
 add_dependencies(uprotocolsdk proto)
 add_dependencies(uprotocolsdk_shared proto)
 
-add_subdirectory(test)
\ No newline at end of file
+add_subdirectory(${CMAKE_SOURCE_DIR}/src/test)
```

Comment:
Avoid this change. keep the original line.

Reply:
Done